client/__tests__/e2e/e2e.test.js

```diff
@@ -518,4 +520,18 @@ test("lasso moves after pan", async () => {
 
   expect(panCount).toBe(initialCount);
 });
+
+const conditionalDescribe =
```

Comment:
this `conditionalDescribe` will skip the describe block if `TEST_AUTH_INTEGRATION` is NOT `"true"`, which is needed since we only want to run the integration tests in `single-cell-infra` GH flow!